cmp/example_test.go

```diff
@@ -147,6 +147,7 @@ func ExampleOption_equalEmpty() {
 	// false
 }
 
+
```

Comment:
Is this change intentional?

It looks like it makes the file no longer `gofmt`ed.